docs/user-guide/work-packages/README.md

```diff
@@ -29,13 +29,13 @@ Work packages can be displayed in a projects timeline, e.g. as a milestone or a
 
 | Topic                                                        | Content                                                      |
 | ------------------------------------------------------------ | :----------------------------------------------------------- |
-| [Work packages views](work-package-views)                    | What is the difference between the work packages views: list view, split screen view, details view? |
-| [Create a work package](create-work-package)                 | How to create a new work package in OpenProject?             |
-| [Edit work package](edit-work-package)                       | How to edit a work package in OpenProject?                   |
-| [Copy, move, delete](copy-move-delete)                       | How to copy, move, delete a work package?                    |
-| [Work package table configuration](work-package-table-configuration) | How to configure the work package table (columns, filters, group by, etc.)? |
-| [Exporting](exporting)                                       | How to export work packages for other tools such as Microsoft Excel? |
-| [Work package relations and hierarchies](work-package-relations-hierarchies) | How to create work package relations and hierarchies?        |
+| [Work packages views](#work-package-views)                    | What is the difference between the work packages views: list view, split screen view, details view? |
+| [Create a work package](#create-work-package)                 | How to create a new work package in OpenProject?             |
+| [Edit work package](#edit-work-package)                       | How to edit a work package in OpenProject?                   |
+| [Copy, move, delete](#copy-move-delete)                       | How to copy, move, delete a work package?                    |
+| [Work package table configuration](#work-package-table-configuration) | How to configure the work package table (columns, filters, group by, etc.)? |
+| [Exporting](#exporting)                                       | How to export work packages for other tools such as Microsoft Excel? |
+| [Work package relations and hierarchies](#work-package-relations-hierarchies) | How to create work package relations and hierarchies?        |
 
```

Comment:
The # in front of the links need to be removed since they don't link to a bookmark on the same page but to a child page.